packages/pointer_interceptor/pointer_interceptor/README.md

```diff
@@ -14,7 +14,7 @@ The result is that Flutter widget's `onTap` (and other) handlers won't fire as e
 
 |The problem...|
 |:-:|
-|![Depiction of problematic areas](https://raw.githubusercontent.com/flutter/packages/main/packages/pointer_interceptor/doc/img/affected-areas.png)|
+|![Depiction of problematic areas](https://raw.githubusercontent.com/flutter/packages/main/packages/pointer_interceptor/pointer_interceptor/doc/img/affected-areas.png)|
```

Comment:
Here and below, we should replace `main` with a pin (the current head, `cb6dbcdd230528c0c246c81d93386c512f9a23d0`, should work fine).